include/common/aarch32/el3_common_macros.S

```diff
@@ -195,6 +195,18 @@
 	cps	#MODE32_mon
 	isb
 
+#if SP_MIN_WITH_SECURE_FIQ
+	/* -----------------------------------------------------
+	 * FIQs are secure interrupts trapped by Monitor and non
+	 * secure is not allowed to mask the FIQs.
+	 * -----------------------------------------------------
+	 */
+	ldcopr	r0, SCR
+	orr	r0, r0, #SCR_FIQ_BIT
+	bic	r0, r0, #SCR_FW_BIT
+	stcopr	r0, SCR
+#endif
+
```

Comment:
right, my fault, sorry. I see 2 ways of implementing this, which do you prefer ?
- either add the sequence at 2 places in bl32/sp_min/aarch32/entrypoint.S,
in `sp_min_entrypoint` and `sp_min_warm_entrypoint`.
- or condition this to `#if defined(IMAGE_BL32) && SP_MIN_WITH_SECURE_FIQ`